website/static/js/pages/shareEmbedConfig.js

```diff
@@ -0,0 +1,7 @@
+var pym = require('pym.js');
```

Comment:
@erinspace To add it to `package.json` run:
`npm install pym.js --save` or `bower install pym.js --save`